test/execution_test.ts

```diff
@@ -781,7 +781,7 @@ describe('Execution', () => {
           await executeFormulaOrSyncFromCLI({
             vm,
             formulaName: 'Students:validateParameters',
-            params: ['Personal'],
+            params: ['', '{"teacher": "Personal", "__brand": "MetadataContext"}'],
```

Comment:
Can you remove the `__brand` here (casting as necessary)? The idea of a brand is that it exists only in the type system, not at runtime.